crates/ruff/src/args.rs

```diff
@@ -501,6 +502,9 @@ pub struct ServerCommand {
     /// Enable preview mode; required for regular operation
     #[arg(long)]
     pub(crate) preview: bool,
+    /// Set the number of background threads used for running tasks concurrently - the default is 4.
```

Comment:
You can omit ` - the default is 4.` This is one of the benefits of encoding it in Clap -- it's already handled in a consistent way:

```
❯ cargo run -- server --help
   Compiling ruff_python_ast v0.0.0 (/Users/crmarsh/workspace/ruff/crates/ruff_python_ast)
   Compiling ruff v0.3.2 (/Users/crmarsh/workspace/ruff/crates/ruff)
   Compiling ruff_python_parser v0.0.0 (/Users/crmarsh/workspace/ruff/crates/ruff_python_parser)
   Compiling ruff_python_literal v0.0.0 (/Users/crmarsh/workspace/ruff/crates/ruff_python_literal)
   Compiling ruff_python_semantic v0.0.0 (/Users/crmarsh/workspace/ruff/crates/ruff_python_semantic)
   Compiling ruff_python_index v0.0.0 (/Users/crmarsh/workspace/ruff/crates/ruff_python_index)
   Compiling ruff_python_codegen v0.0.0 (/Users/crmarsh/workspace/ruff/crates/ruff_python_codegen)
   Compiling ruff_python_formatter v0.0.0 (/Users/crmarsh/workspace/ruff/crates/ruff_python_formatter)
   Compiling ruff_linter v0.3.2 (/Users/crmarsh/workspace/ruff/crates/ruff_linter)
   Compiling ruff_workspace v0.0.0 (/Users/crmarsh/workspace/ruff/crates/ruff_workspace)
   Compiling ruff_server v0.2.2 (/Users/crmarsh/workspace/ruff/crates/ruff_server)
    Finished dev [unoptimized + debuginfo] target(s) in 12.36s
     Running `target/debug/ruff server --help`
Run the language server

Usage: ruff server [OPTIONS]

Options:
      --preview            Enable preview mode; required for regular operation
      --threads <THREADS>  Set the number of background threads used for running tasks concurrently - the default is 4 [default: 4]
  -h, --help               Print help

Log levels:
  -v, --verbose  Enable verbose logging
  -q, --quiet    Print diagnostics, but nothing else
  -s, --silent   Disable all logging (but still exit with status code "1" upon detecting diagnostics)

Global options:
      --config <CONFIG_OPTION>  Either a path to a TOML configuration file (`pyproject.toml` or `ruff.toml`), or a TOML `<KEY> = <VALUE>` pair (such as you might find in a `ruff.toml` configuration file) overriding a specific
                                configuration option. Overrides of individual settings using this option always take precedence over all configuration files, including configuration files that were also specified using `--config`
      --isolated                Ignore all configuration files
```